app/_src/mesh/features/provenance-verification-images.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Verify Build Provenance for Signed Kong Mesh Images
+badge: enterprise
+---
+
+Starting with 2.7.4, {{site.mesh_product_name}} produces build provenance for docker container images, which can be verified using `cosign` / `slsa-verifier` with attestations published to a Docker Hub repository.
+
+This guide provides steps to verify build provenance for signed {{site.mesh_product_name}} Docker container images in two different ways:
+
+* A minimal example, used to verify an image without leveraging any annotations
+* A complete example, leveraging optional annotations for increased trust
+
+For the minimal example, you only need a Docker manifest digest and a GitHub repo name.
+
+{:.important .no-icon}
+> The Docker manifest digest is required for build provenance verification. The manifest digest can be different from the platform specific image digest for a specific distribution.
+
+For the complete example, you need the same details as the minimal example, as well as any of the optional annotations you wish to verify:
+
+| Shorthand | Description | Example Value |
+|---|---|---|
+| `<repo>` | GitHub repository | `kong-mesh` |
+| `<workflow name>` | GitHub workflow name | `build-test-distribute` |
+| `<workflow trigger>` | Github workflow trigger name | `push` |
+| `<version>` | Artifact version to download | `2.7.4` |
```

Comment:
Use `page.version`

Reply:
I am using the `{{page.kong_latest.version}}` everywhere instead `{{ page.latest_version }}`  / `{{page.version}}`which doesn't seem to work. Not sure why though.

Reply:
using `{{page.version}}` to test out the links and examples